about/yocto-wpe.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: post
+title: "Try WPE with Yocto images for Raspberry Pi"
```

Comment:
I think this title should be shortened a bit, otherwise it does not look great on the page, splitting "Pi" to a new line:

![image](https://user-images.githubusercontent.com/2197259/211510654-2101981d-f8c1-49b5-a669-74c2f8cd687d.png)

How about just "Yocto images for the Raspberry Pi"?

Reply:
OK to shortening but maybe we should keep WPE for more relevant web searches?  (I'm not a SEO or anything :stuck_out_tongue_closed_eyes: , but intuitively, should be better)

Like "WPE Yocto images for Raspberry Pi".

Reply:
Oh! I see we have another section titled "Try WPE with Balena", so my suggestion would be inconsistent... maybe we can omit the "Yocto" word in the title and just leave "Try WPE with images for Raspberry Pi"? (after all, whether the images are created with yocto or something else is an implementation detail).